nss-day-cohort-47/reactive-nutshell-the-fantastic-404s
# Lm checkbox function

# Description
Checkbox should now set to completed and remove from list. 

Fixes # (issue): checkbox function working

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Please check tasks to make sure it's only appearing for logged in user and completed tasks are not showing on list.

# Checklist:
- [x] I have added test instructions that prove my fix is effective or that my feature w
